main.js

```diff
@@ -46,15 +46,22 @@ function createNewIdea() {
 // May refactor to:
   // var newIdea = new Idea(titleInput.value, bodyInput.value)
   newIdea.saveToStorage();//this calls localStorage
-  list.push(newIdea);//pushes to array
+  // list.push(newIdea);//pushes to array
   displayCard()
   clearInputs()
 }
 
 function displayCard() {
```

Comment:
Refactor proposal: See below for new pullFromStorage() function:
  pullFromStorage()
  ideaSection.innerHTML = '';
  for(var i = 0; i < list.length; i++) {
    var starImg;
    var starAlt;
    if (list[i].star) {
      starImg = 'assets/star-active.svg'
      starAlt = 'unfavorite this idea'
    } else {
      starImg = 'assets/star.svg'
      starAlt = 'favorite this idea'
    }
  ideaSection.innerHTML += `
    <article class='idea-cards' id="${list[i].id}">
      <header>
        <img src="${starImg}" id="star" alt="${starAlt}">
        <img src="assets/delete.svg" id="delete" alt="delete">
      </header>
      <div class='card-body'>
        <h3>${list[i].title}</h3>
        <p class='card-text'>${list[i].body}</p>
      </div>
      <footer><img src="assets/comment.svg" alt="comment">comment</footer>
    </article>`
  }

Reply:
Thanks for showing me this! And putting it together.